tools/interactive/interactivetool_radiant.xml

```diff
@@ -0,0 +1,75 @@
+        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="GALAXY_WEB_PORT">8080</environment_variable>
+        <environment_variable name="GALAXY_URL">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="API_KEY" strip="True">
+            #if $__user__:
+                #for $api_key in $__user__.api_keys:
+                    ${api_key.key}
+                    #break
+                #end for
+            #end if
+        </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
+    </environment_variables>
+    <command><![CDATA[
+
+        /usr/bin/shiny-server.sh
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="tabular,csv" label="tabular or csv file"/>
+    </inputs>
+    <outputs>
+        <!--data name="outfile" format="txt" /-->
+    </outputs>
+    <tests>
+    </tests>
+    <help>
+<![CDATA[
+
+
+
+
+`Radiant <https://radiant-rstats.github.io/docs/>`_ is an open-source platform-independent browser-based interface for business analytics in R. The application is based on the Shiny package and can be run locally or on a server. Radiant was developed by Vincent Nijs. Please use the issue tracker on GitHub to suggest enhancements or report problems: https://github.com/radiant-rstats/radiant/issues. For other questions and comments please use radiant@rady.ucsd.edu.
+Key features
+
+- Explore: Quickly and easily summarize, visualize, and analyze your data
+- Cross-platform: It runs in a browser on Windows, Mac, and Linux
+- Reproducible: Recreate results and share work with others as a state file or an Rmarkdown report
+- Programming: Integrate Radiant’s analysis functions with your own R-code
+- Context: Data and examples focus on business applications
+
+
+.. class:: infomark
+
+Example input file (TAB separated)::
+
+  "name"	"longitude"	"latitude"	"countryCode"
+  Accipiter striatus Vieillot, 1808	-60.291838	46.328137	CA
+  Accipiter striatus Vieillot, 1808	-114.58927	35.022485	US
+  Accipiter striatus Vieillot, 1808	-93.37406	30.00586	US
+  Accipiter striatus Vieillot, 1808	-79.336288	43.682218	CA
+  Accipiter striatus Vieillot, 1808	-109.156024	31.904185	US
+  Accipiter striatus Vieillot, 1808	-71.098031	42.297408	US
+  Accipiter striatus Vieillot, 1808	-110.927215	32.18203	US
+
+]]>
+    </help>
+    <citations>
+        <!--citation type="doi">10.1111/2041-210X.12945</citation-->
```

Comment:
citaton?

Reply:
idem, I have to remove the entire section